### PR TITLE
streamline docs a bit, and remove redundant info

### PR DESCRIPTION
**Pull request by jalvz**

### CODE REVIEW

**Comment by jalvz** [comment]
@dedemorton do you think you have time for a editorial review of this?

I am not intending to close #1090 , but at least bring it a bit closer.

File: docs/configuration-process.asciidoc [hunk 1]
```diff
@@ -1,31 +1,6 @@
 [[configuration-process]]
 == General configuration options
```
Comment (dedemorton) [inline docs/configuration-process.asciidoc]: Curious as to why you decided to delete the config example. Broken yaml is a common problem (at least for Beats users...not sure about APM). Showing a config example helps users understand the structure of the file. At the very least, I'd provide examples where you describe the option.

File: docs/configuration-process.asciidoc [hunk 2]
```diff
@@ -96,20 +71,19 @@ Disabled by default.
...
 
 [[capture_personal_data]]
 [float]
 ==== `capture_personal_data`
-If set to true,
-APM Server augments data received by the agent with the original IP of either the backend server,
-or the IP and User Agent of the real user (only for real user monitoring).
-It defaults to true.
+If true,
+APM Server augments data received with the original IP of either the backend server or the IP and User Agent of the real user (RUM).
```
Comment (dedemorton) [inline docs/configuration-process.asciidoc]: This sentence is still a little hard to follow.

File: docs/errors.asciidoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+information about the original `exception` that occured
+or about a `log` created when the exception occured.
 
-Both the captured `exception` and the captured `log` of an error can contain `stack trace` information,
-helpful for debugging an error. 
+Both the captured `exception` and the captured `log` of an error can contain a `stack trace`,
+helpful for debugging.
 
-The `culprit` of an error gives some information about the origin of the error. 
+The `culprit` of an error indicate where is originated.
```
Comment (dedemorton) [inline docs/errors.asciidoc]: indicate > indicates

File: docs/errors.asciidoc [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 
-An error can be mapped to the <<transactions,transaction>> during which it happened, 
+An error can be related to the <<transactions,transaction>> during which it happened,
```
Comment (dedemorton) [inline docs/errors.asciidoc]: you might want to remove the passive voice here.

File: docs/exploring-es-data.asciidoc [hunk 5]
```diff
@@ -21,51 +22,42 @@ Default APM `template` and `indices`:
...
-Querying documents that have been collected with a specific APM Server version:
+To query all documents collected with a specific APM Server version:
 ["source","sh",subs="attributes"]
 ------------------------------------------------------------
 GET apm-{version}-*/_search
 ------------------------------------------------------------
 // CONSOLE
 
-If you are only interested in specific document types, e.g. error documents you can use the type in your query:
+To query a specific type, e.g. transactions:
```
Comment (dedemorton) [inline docs/exploring-es-data.asciidoc]: I'd use "for example" rather than "e.g." because non native English speakers might not know what e.g. means.

File: docs/guide/index.asciidoc [hunk 6]
```diff
@@ -7,22 +7,18 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
...
-APM is an application performance monitoring system built on the Elastic Stack.
-It uses Elasticsearch as its data store and allows you to monitor software services and applications in real time.
-Regardless of whether your code is best described as a service or as an application,
-this guide will indiscriminately use the word service.
-
-With APM you can automatically collect detailed performance information from inside your service with only minor changes to your code.
-For example APM agents can measure the response time for incoming requests,
-time spent in database queries,
+Elastic APM is an application performance monitoring system built on the Elastic Stack.
+It allows you to monitor software services and applications in real time (this guide will indiscriminately use the word service),
```
Comment (dedemorton) [inline docs/guide/index.asciidoc]: The parenthetical statement here disrupts the flow of this sentence too much. Maybe add it as a note instead of parenthetically?

File: docs/guide/index.asciidoc [hunk 7]
```diff
@@ -39,30 +35,24 @@ image::apm-architecture.png[Architecture of Elastic APM]
...
 calls to caching servers,
 and HTTP libraries for requests to external services.
 Additionally,
-agents provide an API that you can use to manually instrument anything else you find interesting.
+agents provide an API to manually instrument anything else you find interesting.
 
 *APM Server* is an open source application written in Go which typically runs on dedicated servers.
-It listens on port 8200 by default and receives data from agents periodically through a JSON API.
-APM Server creates documents from the data received from agents and stores them in an Elasticsearch cluster.
+It receives data from the agents through a JSON API, process it, and stores it in Elasticsearch.
```
Comment (dedemorton) [inline docs/guide/index.asciidoc]: You're introducing a couple of unclear antecedents in this sentence. I think the original version is easier to parse (though I agree it's wordy). I'd keep this as a couple of sentences, but tighten up the wording a bit and remove extraneous info.

File: docs/guide/index.asciidoc [hunk 8]
```diff
@@ -86,31 +75,28 @@ The following sections show how to get started quickly with Elastic APM on a loc
 
 First, https://www.elastic.co/downloads/apm/apm-server[download APM Server] for your operating system and extract the package.
 
-In a production environment,
-you would put APM Server on it's own machines,
+In a production environment you would put APM Server on it's own machines,
```
Comment (dedemorton) [inline docs/guide/index.asciidoc]: it's > its

File: docs/guide/index.asciidoc [hunk 9]
```diff
@@ -86,31 +75,28 @@ The following sections show how to get started quickly with Elastic APM on a loc
...
 similar to how you run Elasticsearch.
 You _can_ run it on the same machines as Elasticsearch,
 but this is not recommended,
 as the processes will be competing for resources.
 
-To start APM Server,
-run:
+To start APM Server run:
```
Comment (dedemorton) [inline docs/guide/index.asciidoc]: Commas are usually recommended after introductory phrases of 4 or more words.

File: docs/guide/index.asciidoc [hunk 10]
```diff
@@ -206,15 +189,14 @@ and dashes (must match `^[a-zA-Z0-9 _-]+$`).
...
-simply install the elastic-apm-node module from npm in your service:
+Install the elastic-apm-node module from npm in your service:
 
 [source,bash]
 ----------------------------------
 npm install elastic-apm-node --save
 ----------------------------------
 
-Then configure the elastic-apm-node module inside your service by adding the following lines to the very top of your code:
+Then configure the elastic-apm-node module by adding the following lines to the very top of your code:
```
Comment (dedemorton) [inline docs/guide/index.asciidoc]: I'd remove "very" from "very top" here.

File: docs/rum.asciidoc [hunk 11]
```diff
@@ -2,51 +2,43 @@
...
+To use it you need to <<rum-enable,enable RUM support>> in the APM Server.
 --
 
 [[sourcemaps]]
 == Source Maps
-It is common practice to minify JavaScript code for several reasons, e.g. performance gain. 
-This can make debugging very difficult, as it is hard to read the minified files.
-Source mapping can help for debugging minfied JavaScript files, 
-by mapping code from the minified files to the original source code. 
+It is common practice to minify JavaScript code, e.g. to reduce network latency.
```
Comment (dedemorton) [inline docs/rum.asciidoc]: again I would avoid using e.g. I don't have a strong preference personally, but it's been drilled into me by editors over the years. :-)

File: docs/rum.asciidoc [hunk 12]
```diff
@@ -2,51 +2,43 @@
...
 
-If multiple source maps with the same meta information are found, 
-the source map with the latest upload timestamp is used. 
+If multiple source maps are found,
+the one with the latest upload timestamp is used.
 
-In case a matching source map is found and the source map can be applied to the `stack trace frame`, 
-the frame's information is updated with the mapped information before the record is indexed.
-The following information is changed to reflect the original source code file, when source mapping is applied:
+If succeeded, the following attributes in the `stack trace frames` are overwritten:
```
Comment (dedemorton) [inline docs/rum.asciidoc]: "If succeeded" is too vague here. What do you mean by succeeded?

File: docs/security.asciidoc [hunk 13]
```diff
@@ -4,19 +4,18 @@
...
 ==== Secret token
 
-You can configure a secret token which is sent with every request from the APM agents to the server.
-This string is used to ensure that only your agents can send data to your APM servers.
+You can configure a secret token to authorize requests to the APM Server,
+and ensure that only your agents can send data to your APM servers.
 Both the agents and the APM servers have to be configured with the same secret token.
 
-NOTE: The usage of a secret token only provides any security when used in combination with having SSL/TLS configured.
+NOTE: The usage of a secret token only provides any security when used in combination with having SSL/TLS configured,
```
Comment (dedemorton) [inline docs/security.asciidoc]: You could probably condense this a bit and say something like: 

Secret tokens provide security only when used in combination with SSl/TLS. 

Do you mean that the whole statement is not applicable to RUM? Not sure given the current structure of your sentence, but you'll have to work that in...maybe in a second sentence.

File: docs/setting-up-and-running.asciidoc [hunk 14]
```diff
@@ -26,7 +26,7 @@ You can change the defaults by supplying a different address on the command line
 
 [[apm-server-configuration]]
 === Configuration file
-To configure APM Server, you can also update the `apm-server.yml` configuration file.
+To configure APM Server you can also update the `apm-server.yml` configuration file.
```
Comment (dedemorton) [inline docs/setting-up-and-running.asciidoc]: Typically we do use commas after intro phrases of 4 or more words.

File: docs/sourcemap-api.asciidoc [hunk 15]
```diff
@@ -1,14 +1,12 @@
...
-helpful if you are using <<rum, real user monitoring (RUM)>>.
+The APM Server exposes an API endpoint to upload source maps for <<rum, real user monitoring (RUM)>>.
 
 [[sourcemap-endpoint]]
 [float]
-=== Upload Endpoint 
-To upload a source map you need to send a `HTTP POST` request
-with `Content-Type` set to `multipart/form-data` to the APM Server source maps endpoint:
+=== Upload endpoint
+Send a `HTTP POST` request with `multipart/form-data` `Content-Type`  to the source map endpoint:
```
Comment (dedemorton) [inline docs/sourcemap-api.asciidoc]: minor: extra space after `Content-Type`.

Why not wrap all of this in back ticks? Is something missing here (maybe "and" to connect the two things)? `multipart/form-data` `Content-Type`

File: docs/sourcemap-api.asciidoc [hunk 16]
```diff
@@ -18,26 +16,19 @@ http(s)://{hostname}:{port}/v1/rum/sourcemaps
...
-https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k[Source map revision 3 proposal].
-
+The source map must follow the
+https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k[Source map revision 3 proposal] spec and be attached as a `file upload`.
 
 [[sourcemap-api-examples]]
 [float]
 ==== Example
 
-Send an example source map to the APM Server:
```
Comment (dedemorton) [inline docs/sourcemap-api.asciidoc]: Is this text not accurate? IMO, it's nice to introduce examples for people who scan stuff quickly.

File: docs/storage-management.asciidoc [hunk 17]
```diff
@@ -3,73 +3,65 @@
...
 
 [[reduce-sample-rate]]
 [float]
 === Reduce the sample rate
-In case you are monitoring high traffic services you might want to decrease the number of transactions that are sampled. 
-The transaction sample rate directly influences the number of documents to be indexed
-and therefore is the most obvious way to reduce storage. 
 
-The transaction sample rate is controlled in the configuration of agents (for example for {apm-py-ref}/configuration.html#config-transaction-sample-rate[Python] and {apm-node-ref}/agent-api.html#transaction-sample-rate[Node.js]).
+The transaction sample rate directly influences the number of documents (more precisely, Spans) to be indexed
```
Comment (dedemorton) [inline docs/storage-management.asciidoc]: I don't think "spans" should be capitalized. Typically we capitalize product names, not product terminology or features.

File: docs/storage-management.asciidoc [hunk 18]
```diff
@@ -3,73 +3,65 @@
...
+and therefore is the most obvious way to reduce storage.
 
-Reducing the transaction sample rate does not affect the collection of metrics such as _Transactions Per Second_.
+The transaction sample rate is a configuration of the agents.
```
Comment (dedemorton) [inline docs/storage-management.asciidoc]: This sentence isn't clear. Is there a word missing? Do you mean it's a configuration setting?

File: docs/storage-management.asciidoc [hunk 19]
```diff
@@ -3,73 +3,65 @@
...
-you can either delete data periodically with a tool like {curator-ref-current}[Curator] 
-or by using the {ref}/docs-delete-by-query.html[Delete By Query API]
-or by using the {kibana-ref}/managing-indices.html[Kibana Index Management UI]. 
+
+You might want to keep data only for a defined time period and delete old documents periodically,
+or you might also want to delete data collected for specific services or customers,
+or delete specific indices.
+Depending on your use case,
+you can either delete data periodically with a tool like {curator-ref-current}[Curator],
+using the {ref}/docs-delete-by-query.html[Delete By Query API]
```
Comment (dedemorton) [inline docs/storage-management.asciidoc]: "by using" is clearer in both instances here, IMO, because you avoid the dangling modifier problem.

File: docs/storage-management.asciidoc [hunk 20]
```diff
@@ -107,8 +99,8 @@ INFO      "delete_indices" action completed.
 [float]
 ==== Delete data matching a query
 
-In case you want to delete documents matching a specific query, e.g. all documents with a given `context.service.name`,
-you can do this by sending the following request:
+To delete documents matching a specific query, e.g. all documents with a given `context.service.name`,
```
Comment (dedemorton) [inline docs/storage-management.asciidoc]: Suggested edit:

To delete documents matching a specific query, for example all documents with a given `context.service.name`, use the following request:

File: docs/transaction-api.asciidoc [hunk 21]
```diff
@@ -142,7 +134,7 @@ curl http://localhost:8200/v1/transactions \
   --data @docs/data/intake-api/generated/transaction/payload.json
 ------------------------------------------------------------
 
-See examples on how a transaction request to the APM Server can look like:
+See how transactions look like:
```
Comment (dedemorton) [inline docs/transaction-api.asciidoc]: This sentence seems a little awkward. Could you just say "Example transaction requests"? I think it's OK to use an incomplete sentence to introduce an example.

Sorry, don't have time to look closely at the context here to see if my edit makes sense.

File: docs/transactions.asciidoc [hunk 22]
```diff
@@ -1,48 +1,40 @@
 [[transactions]]
 == Transactions
 
-A transaction represents one event, captured by an Elastic APM agent within one service. 
-A transaction can for example be a single HTTP request or an asynchrounous background job within one service.
+A transaction describes an event captured by an Elastic APM agent instrumeting a service.
+A transaction can for example be a HTTP request or an asynchrounous background job.
```
Comment (dedemorton) [inline docs/transactions.asciidoc]: an HTTP request.

File: docs/transactions.asciidoc [hunk 23]
```diff
@@ -1,48 +1,40 @@
...
 === Spans
-Several spans can be related to a transaction. 
-The relationship is indicated by a `transaction.id`.
-
-A span contains information about a specific code path, 
-executed as part of a transaction. 
-Elastic APM agents automatically instrument a variety of libraries, 
-but also support custom instrumentation for code paths.
-Every code path that is captured by an agent creates a span.
+Spans can have 0, 1 or many spans. Spans have a `transaction.id` attribute that refer to their transaction.
```
Comment (dedemorton) [inline docs/transactions.asciidoc]: minor: Just an FYI that use the oxford comma at Elastic: So this would be "0, 1, or many spans"

File: docs/transactions.asciidoc [hunk 24]
```diff
@@ -1,48 +1,40 @@
...
+a span describing the database query will be created.
 In such a case the name of the span will contain information about the query itself,
-and the type will hold information about the database type. 
+and the type about the database.
+
+Typically, agents instrument automatically a variety of libraries,
+but also provide an API for ad hoc instrumentation of specific code paths.
 
-A span can also contain `stack trace` information.
+Transactions are stored in <<transaction-indices, transaction indices>>and spans are stored in separated indices by default.
```
Comment (dedemorton) [inline docs/transactions.asciidoc]: Needs a space after the closing  >>

File: docs/exploring-es-data.asciidoc (hunk 5 above)
Comment (jalvz) [reply docs/exploring-es-data.asciidoc]: e.g. is a latin voice, so all romanic languages speakers know it, and i think germanic and slavic speakers  know it too...
there are ofc more languages than those, so if you think is an actual issue i'm happy to change it. we have a lot of e.gs spread all over the place.

File: docs/configuration-process.asciidoc (hunk 1 above)
Comment (jalvz) [reply docs/configuration-process.asciidoc]: it didn't seem too informative, as is basically a snippet copied from `apm-server.yml`
i was actually thorn on whether to remove it or not, so if you think is better to have it ill add it back, but probably smaller and showing how yml configs can look like (nested attrs, dotted attrs, etc.)

File: docs/guide/index.asciidoc (hunk 9 above)
Comment (jalvz) [reply docs/guide/index.asciidoc]: "introductory phrase" means the ones starting with  `to, if, when, for, etc.`?

File: docs/rum.asciidoc (hunk 11 above)
Comment (jalvz) [reply docs/rum.asciidoc]: ah, i guess this answers my comment https://github.com/elastic/apm-server/pull/1169#discussion_r205080708
:)

ill change it then 👍

File: docs/security.asciidoc (hunk 13 above)
Comment (jalvz) [reply docs/security.asciidoc]: yeah exactly, ill rephrase

File: docs/storage-management.asciidoc (hunk 17 above)
Comment (jalvz) [reply docs/storage-management.asciidoc]: right. 
same rule applies eg if it was a link to a section (something like <<spans-section, spans>>?

File: docs/storage-management.asciidoc (hunk 18 above)
Comment (jalvz) [reply docs/storage-management.asciidoc]: yeah. i was thinking the same and couldn't find what was the word missing. thanks :)

**Comment by jalvz** [comment]
Many thanks for the thorough review!
Tried to address comments here: d714989c9972508ff38b8652b52775476426ccc9

Another question: do you have a rule in beats for line breaks? doesn't have an impact in rendering, but or docs are a bit uncomfortable to read with some very long lines and some very short lines...

File: docs/configuration-process.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/configuration-process.asciidoc]: yah, a truncated example would be fine...just something that shows structure so users know  where options are defined (whether they need to be namespaced, at the root, etc).

File: docs/guide/index.asciidoc (hunk 9 above)
Comment (dedemorton) [reply docs/guide/index.asciidoc]: Typically the introductory phrase is followed by a comma then a noun, but in this case the noun "you" is implied. TBH, I think your sentence is OK because it's easy to parse. I just wanted to mention the general rule because I wouldn't want you to spend time removing commas that are OK.

File: docs/storage-management.asciidoc (hunk 17 above)
Comment (dedemorton) [reply docs/storage-management.asciidoc]: If it's capitalized, I would expect the link text to match the title of the page you're pointing to. Generally, it's better to let asciidoc resolve the link text (unless you're sure that whatever text you provide is stable and won't need to change over time). Lowercase is just fine if the link text is meant to be descriptive.

**Comment by dedemorton** [comment]
@jalvz In Beats, we try to wrap at 80 characters (though we're not consistent, especially in older topics that existed before we had a convention).

I don't personally like the varying line lengths in the APM doc source. TBH, I can't remember why APM settled on that convention, but I remember it was some kind of best practice. It doesn't work well for me.

In general, when I update content, I try to limit the diffs in GitHub to make it easier for people to review changes. Sometimes I'll make a conscious effort to clean up a topic (especially if there are no line wraps), but I prefer to leave the line breaks as-is unless I'm adding or removing content.

It would be nice if we could get a full-time writer on the team before embarking on #1170 because I think that person will have some good ideas and add a fresh perspective.
